front/src/containers/Notification/style.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+import { StyleSheet } from 'react-native'
+
+export const GlobalStyle = StyleSheet.create({
+  Icon: {
+    zIndex: 1,
+    position: 'absolute',
+    left: 65,
+    top: 60,
+  }
+})
+
+export const Container = styled.SafeAreaView`
+  flex: 1;
```

Comment:
This isn't needed, the height of component view, resize according with child component. So, you can use only a view in render